src/BingChat/BingChatClient.cs

```diff
@@ -68,7 +68,8 @@ public async Task<BingChatConversation> CreateConversation()
         if (cookies.Count == 0)
             cookies.Add(new Uri("https://www.bing.com"), new Cookie("_U", Utils.GenerateRandomHexString()));
 
-        using var handler = new HttpClientHandler { CookieContainer = cookies };
+        using var handler = new HttpClientHandler
+            { CookieContainer = cookies, Proxy = _options.Proxy, UseProxy = _options.Proxy != null };
```

Comment:
By the way, according to documentation, you do not need to set `UseProxy` property because it is true by default and this change would alter the behavior.

Ref: https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclienthandler.useproxy